test/integration/backward_compatible/topic/ReliableTopicOnClusterRestart.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const chai = require('chai');
+chai.should();
+
+const RC = require('../../RC');
+const { Client } = require('../../../../');
+const { promiseWaitMilliseconds, assertTrueEventually, randomString } = require('../../../TestUtil');
+
+describe('ReliableTopicOnClusterRestartTest', function () {
+
+    let cluster;
+    let member;
+    let client1;
+    let client2;
+
+    before(async function () {
+        cluster = await RC.createCluster(null, null);
+        member = await RC.startMember(cluster.id);
+    });
+
+    beforeEach(async function () {
+        client1 = undefined;
+        client2 = undefined;
+    });
+
+    afterEach(async function () {
+        if (client1) await client1.shutdown();
+        if (client2) await client2.shutdown();
+    });
+
+    after(async function() {
+        await RC.shutdownCluster(cluster.id);
+    });
+
+    it('should continue on cluster restart when data lost and after invocation timeout', async function () {
+        const invocationTimeoutMillis = 2000;
+
+        client1 = await Client.newHazelcastClient({
```

Comment:
I think we can move client construction to a function to eliminate duplicate code